46.md

```diff
@@ -1,98 +1,174 @@
 
-Events sent by the client to the remote signer have the following format:
+Clients can subscribe to events of `kind: 24136`, `kind: 24137`, and `kind: 24138` that have an "e" tag containing the ID of their request event to listen for the response, auth_challenge, or error to their command.
 
-```js
+```json
 {
-  "pubkey": "<client-key-hex>"
-  "kind": 24133,
-  "tags": [
-    ["p", "<signer-key-hex>"]
-  ],
-  "content": "nip04_encrypted_json({id: <random-string>, method: <see-below>, params: [array_of_strings]})",
-  ...
+    "id": <id>,
+    "kind": <kind for command>, // Each command has it's own kind
+    "pubkey": <local pubkey of the user>,
+    "content": <nip44(<request>)>,
+    "tags": [["p", <remote pubkey>]],
+    "created_at": <unix timestamp in seconds>,
 }
 ```
 
-And the events the remote signer sends to the client have the following format:
+| Kind    | Command                  | Content field (always NIP-44 encrypted)                                         | Data Type               |
+| ------- | ------------------------ | ------------------------------------------------------------------------------- | ----------------------- |
+| `24140` | `create_account`         | {username: <username>, domain: <domain>, <any other arbitrary key/value pairs>} | JSON stringified object |
+| `24141` | `connect`                | -                                                                               | null                    |
+| `24142` | `sign_event`             | nostr event to sign                                                             | JSON stringified object |
+| `24143` | `ping`                   | -                                                                               | null                    |
+| `24144` | `get_relays`             | -                                                                               | null                    |
+| `24145` | `nip04_encrypt`          | {third_party_pubkey: <third_party_pubkey>, content: <plaintext to encrypt>}     | JSON stringified object |
+| `24146` | `nip04_decrypt`          | {third_party_pubkey: <third_party_pubkey>, content: <ciphertext to decrypt>}    | JSON stringified object |
+| `24147` | `nip44_conversation_key` | third_party_pubkey                                                              | string                  |
+| `24148` | `nip44_encrypt`          | {third_party_pubkey: <third_party_pubkey>, content: <plaintext to encrypt>}     | JSON stringified object |
+| `24149` | `nip44_decrypt`          | {third_party_pubkey: <third_party_pubkey>, content: <ciphertext to decrypt>}    | JSON stringified object |
+
+## Response Event `kind: 24136`
+
+```json
+{
+    "id": <id>,
+    "kind": 24136,
+    "pubkey": <pubkey of the remote signer>,
```

Comment:
```suggestion
    "pubkey": <remote pubkey>,
```

Reply:
This would actually be the pubkey of the remote signer (the bunker) since these response objects are coming from the bunker/remote signer